frontend/src/pages/EventDetailsPage/components/buttons/SignInButton.ts

```diff
@@ -23,4 +24,7 @@ function SignInButton({ eventId, children = 'Sign In' }: SignInButtonProps) {
   });
 }
 
+
+
+
```

Comment:
nit: it's good practice to eliminate the number of unnecessary changes to make code review easier
no need to change - just a small thing